mosip/on-prem/README.md

```diff
@@ -186,11 +186,19 @@ Below contains some of the RKE cluster related operations in brief:
 
 Note: Before adding/removing nodes make sure that ```rke version``` should be same as you have installed during initial cluster creation as mentioned in the Pre-requisites. 
 
-* Adding/Removing nodes to cluster
-  _This step is only required if you have to add/delete nodes to an existing cluster._
+* Adding nodes to cluster
+  _This step is only required if you have to add nodes to an existing cluster._
```

Comment:
_Note_: This step is required only if you have to add nodes to an existing cluster.